scripts/overwatch.lic

```diff
@@ -59,6 +75,13 @@ module OverWatch
   def self.watch
     while (line = get)
       case line
+      # Let's try to keep track of hiders.
+      when HIDING_REGEX
+        OverWatch.track_hidden_targets(XMLData.room_id,line)

```

Comment:
Aren't actually using `line` in `OverWatch.track_hidden_targets`, it should be removed.